packages/dcos-integration-test/extra/test_ucr.py

```diff
@@ -0,0 +1,229 @@
+import uuid
+
+TEST_APP_NAME_FMT = 'integration-test-{}'
+
+
+def test_if_ucr_app_can_be_deployed_with_image_entrypoint(dcos_api_session):
+    """Marathon app deployment integration test using the Mesos Containerizer.
+
+    This test verifies that a marathon ucr app can execute a docker image
+    default entrypoint.
+    """
+    test_uuid = uuid.uuid4().hex
+    app = {
+        'id': TEST_APP_NAME_FMT.format(test_uuid),
+        'cpus': 0.1,
+        'mem': 32,
+        'instances': 1,
+        'container': {
+            'type': 'MESOS',
+            'docker': {
+                'image': 'mesosphere/inky'
+            }
+        }
+    }
+    dcos_api_session.marathon.deploy_and_cleanup(app)
```

Comment:
I don't think this does anything. This method is a context manager. You must do:
```
with dcos_api_session.deploy_and_cleanup(app):
    pass
```
Doing what you did here just returns the _context_, rather you need to *enter* the context.

Furthermore, your app as written should never deploy. The method by default expects a health check to return back successful before it can say the app is deployed. Once you fix this code to actually deploy the container, it will never be able to to. Turning off health checks is not an option in this case because you have no other assertions. Thus, without health checks, the test result would be 100% trivial as the deployment could just fail spectacularly.